### DISCUSSION TITLE
I want to completely delete my website.

### DISCUSSION THREAD

**Discussion by ILoveYouF** [community/community#148078] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I published my website on GitHub Pages some time ago, and now I want to delete it. About a week ago, I completely deleted the repository, but when I search for the website on the internet, the page still appears, although without content (showing 'There isn't a GitHub Pages site here.'). However, the page and some images from it still appear on the internet. I want all of that to be removed from the internet. Please help.

**Comment by aflacake** [comment]
**Deleted repositories same as deleted GitHub Pages.** Any content "Not Found", be wait a moment about internet collet any data. Every search is update, the page will disappear
